build.gradle.kts

```diff
@@ -26,21 +26,19 @@ dependencies {
     implementation("com.natpryce:konfig:1.6.10.0")
 
     implementation("org.slf4j:slf4j-simple:2.0.0-alpha1")
+
+    testImplementation("org.jetbrains.kotlin:kotlin-test-junit:1.4.0")
+}
+
+application {
+    mainClassName = "mainKt"
 }
 
 tasks.withType<Jar> {
     duplicatesStrategy = DuplicatesStrategy.EXCLUDE
 
     configurations["compileClasspath"].forEach { from(zipTree(it.absoluteFile)) }
     configurations["runtimeClasspath"].forEach { from(zipTree(it.absoluteFile)) }
-
-    manifest {
```

Comment:
I'm not sure it would deploy on server without the manifest

Reply:
`manifest/main` is now managed by `MainFile` or smth like that